Looms.PoS/Looms.PoS.Domain/Daos/ProductVariationDao.cs

```diff
@@ -0,0 +1,12 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Looms.PoS.Domain.Daos;
+
+public record ProductVariationDao
+{
+    public Guid Id { get; init; }
+    public Guid ProductId { get; init; }
+    //TODO: in the class diagram this doesnt have price, but in the API specification it does (as optional, but still)
```

Comment:
Add price as it may change from variation to variation

Reply:
fixed in "fixed indentation/layout problems, changed Variation to Variation request and made create and update requests take IEnumerable<VariationRequest>"